tests/lang_experimental/concepts/tconcepts_issues.nim

```diff
@@ -31,7 +31,7 @@ Meow
 joinable: false
 """
 
-import macros, typetraits, os, posix
+import macros, typetraits, os
```

Comment:
Until the test is properly reduced, I think the `posix` import has to stay, and the test needs to be disabled for Windows. Refer to https://github.com/nim-lang/nim/issues/6782, from which the test was taken.

When you do so, could you also add a note stating that the `t6782` case needs further reduction?

Reply:
It might be better if you create an issue for that.

I'll drop this commit from the branch.